app/notifications/validators.py

```diff
@@ -126,6 +127,17 @@ def check_if_service_can_send_files_by_email(service_contact_link, service_id):
         )
 
 
+def error_if_service_using_email_verification_flow_without_permission(file_upload_personalisation, service_permissions):
```

Comment:
Should we test this error? That a service without permission tries to send file with verification and they get the right error message in the response?

Reply:
Not sure how I missed this! Thanks

Reply:
Added 👍